modules/virt-about-container-disks.adoc

```diff
@@ -0,0 +1,32 @@
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/virt-using-containerdisk-volumes.adoc
+// * virt/virtual_machines/importing_vms/virt-importing-container-disk-with-datavolumes.adoc
+
+[id="virt-about-container-disks_{context}"]
+= About container disks 
+
+A container disk is a virtual machine image stored as a container image in a container image registry. Container disks can deliver the same disk images to multiple virtual machines and can be used to create large numbers of virtual machine clones.
+
+A container disk can either be imported into a DataVolume and attached to a virtual machine or attached directly to a virtual machine as an ephemeral `containerDisk` volume.
```

Comment:
The import destination is a PVC, not a DataVolume. 

The DataVolume itself represents the automated workflow of importing an image source "s3, http, registry, etc..." into a PVC. So when we talk about importing, i think the emphasis should be on a DataVolume importing a containerDisk onto a PVC for persistence rather than "importing into a DataVolume"